app/services/migration/migrators/participant_id_change.rb

```diff
@@ -0,0 +1,36 @@
+module Migration::Migrators
+  class ParticipantIdChange < Base
+    class << self
+      def model_count
+        ecf_participant_id_changes.count
+      end
+
+      def model
+        :participant_id_change
+      end
+
+      def dependencies
+        %i[user]
+      end
+
+      def ecf_participant_id_changes
+        Migration::Ecf::ParticipantIdChange
+          .joins(:user)
+          .where(user: { id: User.ecf_users.pluck(:id) })
```

Comment:
do we need to filter this 🤔 I think maybe it's worth importing all of them incase we are hiding errors?

Reply:
I figured it would be best to filter them here as we're only importing a subset of the users so presumed we'd only want the corresponding participant id changes? otherwise we will try and bring across id changes for users we didn't import and it'll fail as a foreign key won't exist

Reply:
ah yes, we don't want all ECF users 👍